tests/functional/src/main/resources/META-INF/mule-module.properties

```diff
@@ -19,6 +19,7 @@ artifact.export.classPackages=org.mule.functional,\
                               org.mule.functional.util.ftp,\
                               org.mule.functional.util.sftp,\
                               org.mule.tck.util,\
+                              org.mule.functional.junit4.runners,\
```

Comment:
They are in different packages, org.mule.functional.junit4.runners only has the runner implementation and the annotations:  delegate, pluginsAware and runnerConfig.
The rest of the classes used here are internal, and they are all in a subpackage of org.mule.functional.classloading.isolation

The maven plugin that validates the exported packages analysed this and there is no need to expose the org.mule.functional.classloading.isolation package and subpackages.